Makefile

```diff
@@ -51,11 +51,11 @@ clean-test: ## remove test and coverage artifacts
 	rm -fr .pytest_cache
 
 lint: ## check style with flake8
-	flake8 landlab --exclude=examples
+	flake8 landlab tests --exclude=examples
```

Comment:
I *think* we can remove the `--exclude=examples` as I removed the examples folder. 

I should have done that on #996.